trace-protobuf/src/pb/stats.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
```

Comment:
Might be missing a copyright disclaimer - is protobof file - code (?)

Reply:
i don't believe the proto files need copyright disclaimers (the proto files in the regular agent don't have them)